src/dmtcp_launch.cpp

```diff
@@ -419,6 +419,13 @@ processArgs(int *orig_argc, const char ***orig_argv)
 int
 main(int argc, const char **argv)
 {
+  // FIXME:  This was changed in Mar., 2022.  We can remove this msg in 2 or 3 years.
+  if (getenv("DMTCP_ABORT_ON_FAILED_ASSERT")) {
+    fprintf(stderr, "*********************************************\n"
```

Comment:
Please use JTRACE here and for that you'll need to move it _after_ `initializeJalib()` call.